### ISSUE TITLE
Linux binnaries

### ISSUE THREAD

**Issue by dimitrov-adrian** [eliecharra/iso2god-cli#8]
Could you add linux binaries as well?

**Comment by montvid** [comment]
Yes wuold be super cool! Now i have to try make it work with wine and it does not always work with wine...
